Formula/groestlcoin.rb

```diff
@@ -0,0 +1,50 @@
+class Groestlcoin < Formula
+  desc "Decentralized, peer to peer payment network"
+  homepage "https://groestlcoin.org/groestlcoin-core-wallet/"
+  url "https://github.com/Groestlcoin/groestlcoin/releases/download/v22.0/groestlcoin-22.0.tar.gz"
+  sha256 "6c412361e7ff84e9f7cf6a25be0d18742098ddc0f91688b368b0be3e6daa46f1"
+  license "MIT"
+  head "https://github.com/groestlcoin/groestlcoin.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+  depends_on "libevent"
+  depends_on macos: :catalina # groestlcoin requires std::filesystem, which is only supported from Catalina onwards.
+  depends_on "miniupnpc"
+  depends_on "sqlite"
```

Comment:
Does this not work like this?
```suggestion
  uses_from_macos "sqlite"
```

Reply:
what about linux users then?

Reply:
`uses_from_macos` essentially means `depends_on  "" if OS.linux?`

Reply:
Seeing how CI is currently failing on this, I'd say there's a fairly strong preference to depend on the system version.

Reply:
Suggestion implemented